src/components/account/Share/Share.tsx

```diff
@@ -104,11 +80,7 @@ export const Share: React.FC = () => {
           </button>
         </div>
         <div className="Share__qr">
-          {qrCode ? (
-            <img src={`data:image/svg+xml;utf8,${encodeURIComponent(qrCode)}`} />
-          ) : (
-            'Loading...'
-          )}
+          <W3mQrCode size={250} imageUrl={ensAvatar ?? '/logo.png'} uri={uri} />
```

Comment:
We could have the width as 318 to prevent extra whitespace on the sides

Reply:
We also have to change styling of the container div to not explicitly provide 250px as the width 
<img width="212" alt="image" src="https://user-images.githubusercontent.com/26746725/232912832-6742bed7-7115-47f9-b131-e5e3270d8a26.png">

Reply:
The `svg` underneath has fixed dimensions